src/inputs/radio.scss

```diff
@@ -5,7 +5,7 @@
 @mixin iui-radio {
   @include iui-inputs-checkbox-radio;
 
-  .iui-radio-dot {
+  .iui-radio-dot::after {
```

Comment:
Not sure if this is supposed to be part of the fix in this pr but I noticed when zooming in a lot sometimes the radio dot becomes an oval. Does this look right? 

![image](https://user-images.githubusercontent.com/88331924/148567308-566b8f33-efd1-4635-a28a-56f8cce50ec1.png)

![image](https://user-images.githubusercontent.com/88331924/148567053-287aac48-4d0d-40aa-9050-6dfa4fb74e06.png)

Reply:
Yikes, that's not good.  Investigating, i think it probably has something to do with `flex-shrink`.

Reply:
Yep, it was `flex-shrink`, change is pushed.  Nice catch.